packages/apps/test/ticTacToe.spec.ts

```diff
@@ -0,0 +1,251 @@
+import * as ethers from "ethers";
+
+import * as Utils from "@counterfactual/test-utils";
+
+const TicTacToe = artifacts.require("TicTacToe");
+
+const StaticCall = artifacts.require("StaticCall");
+
+const web3 = (global as any).web3;
+const { provider, unlockedAccount } = Utils.setupTestEnv(web3);
+
+contract("TicTacToe", (accounts: string[]) => {
+  let game: ethers.Contract;
+
+  const stateEncoding =
+    "tuple(address[2] players, uint256 turnNum, uint256 winner, uint256[3][3] board)";
+
+  beforeEach(async () => {
+    const contract = new ethers.Contract("", TicTacToe.abi, unlockedAccount);
+    TicTacToe.link("StaticCall", StaticCall.address);
+    game = await contract.deploy(TicTacToe.binary);
```

Comment:
If you rebase on `develop` you can instead write 
```ts
const TicTacToe = AbstractContract.loadTruffleArtifact("TIcTacToe", {StaticCall});
game = await TicTacToe.deploy(unlockedAccount);
```

Reply:
Actually why are you redeploying the App before each test? I'd imagine you only need to do that once for the whole file.